### DISCUSSION TITLE
BED file directionality?

### DISCUSSION THREAD

**Discussion by willgarland** [harbourlab/SparK#13] [Q&A]
Hi - first off, love this tool. I'm just putting together a manuscript and it's great to have a nice pipeline to generate nice consistent tracks for figures. 
I was just wondering if there was a way to add directionality to BED files?

**Comment by harbourlab** [comment]
Good morning! I’m happy you find SparK useful! Could you give an example of what exactly you envision? I’m sure I could implement that quickly. Cheers

Sent from my iPhone

On Dec 14, 2020, at 9:56 AM, willgarland <notifications@github.com> wrote:

﻿
CAUTION: This email originated from outside the organization. DO NOT CLICK ON LINKS or OPEN ATTACHMENTS unless you know and trust the sender.


Hi - first off, love this tool. I'm just putting together a manuscript and it's great to have a nice pipeline to generate nice consistent tracks for figures.
I was just wondering if there was a way to add directionality to BED files?

**Comment by willgarland** [comment]
I guess similar to how IGV visualises them with arrows within the bar (as below). It might require a wider line for the BED track in SparK to be able to visualise.

![image](https://user-images.githubusercontent.com/75995523/102104032-01876400-3e2e-11eb-9a12-6a156724b31f.png)
